src/ocean/util/app/DaemonApp.d

```diff
@@ -493,6 +534,74 @@ public abstract class DaemonApp : Application,
         this.unix_socket_ext.initializeSocket(this.epoll);
     }
 
+    /**************************************************************************
+
+        Registers AppStatus instance with the daemon application. This allows
+        DaemonApp to automatically connect unix domain socket to the AppStatus'
+        output.
+
+        Params:
+            app_status = instance of AppStatus to connect the output to.
+
+    **************************************************************************/
+
+    public void registerAppStatus (AppStatus app_status)
```

Comment:
Would there be a way to reverse this? Have a `registerUnixSocket` method in `AppStatus`. It's a bit nasty to integrate this deprecated stuff into `DaemonApp`.

Reply:
Good point.